tests/selenium/src/tests/71-import-delete-favorite-group.mjs

```diff
@@ -5,6 +5,7 @@ import { By } from 'selenium-webdriver';
 import { deleteFavGroup, getFiles, uploadFavorites } from '../util.mjs';
 import actionFinish from '../actions/actionFinish.mjs';
 import actionCheckFileExist from '../actions/actionCheckFileExist.mjs';
+import { TIMEOUT_REQUIRED } from '../options.mjs';
```

Comment:
Please avoid using TIMEOUT_XXX in test's code. They are library-related internal constants, not for using in the tests.